values.yaml

```diff
@@ -363,6 +363,22 @@ prometheus:
   podLabels: {}
   # Data retention
   retention: 15d
+  # Scrape interval
+  scrape_interval: 30s
+  evaluation_interval: 30s
+  velero:
+    scrape_interval: 30s
+  kubeState:
+    scrape_interval: 10s
+  fluentd:
+    scrape_interval: 30s
+  nodeExporter:
+    scrape_interval: 30s
+  airflow:
+    scrape_interval: 10s
+  postgresqlExporter:
+    scrape_interval: 60s
+    scrape_timeout: 30s
```

Comment:
These are probably not worth exposing to both the prometheus chart and the astronomer umbrella chart. I think we can go with just the prometheus values.yaml changes. It'll still work the same.

Reply:
Yes, it works the same, only thing is rather than manually doing
```
--set prometheus.scrape_interval=30s
```
for each helm release will be an effort and error prone, giving here some default will be seamless, i am not sure of exposing these time intervals, who is best person to ask or is it our team to decide?

Reply:
We would not need to use `--set`, we could still use values in `values.yaml` files. EG: [we set `prometheus.ingressNetworkPolicyExtraSelectors` in a values file](https://github.com/astronomer/google-environments/blob/09da774b52/prod/cloud/app/config.yaml#L107-L108) and that is not exposed to [the astronomer umbrella chart's values file](https://github.com/astronomer/astronomer/blob/ec12fba78c/values.yaml). It is exposed only within [the prometheus subchart's values.yaml](https://github.com/astronomer/astronomer/blob/ec12fba78c/charts/prometheus/values.yaml#L62). Using `--set` gives you no extra features than `-f values.yaml`. Setting those values in the astronomer umbrella chart's values file is just overriding whatever is in the subchart's values file, and in this case it is not helpful. The only reason we'd want to put things into the umbrella chart is to give customers additional hints that those values are configurable, so we should be selective about what we put in there. We do not want to drown out the more important values that most customers will want to tune by including a bunch of values that most customers are not going to modify.